### DISCUSSION TITLE
How to set the terminal that appears every time you open the VSCode editor to the right instead of the bottom

### DISCUSSION THREAD

**Discussion by dong-lufei** [microsoft/vscode-discussions#430] [Extension Development QnA] [answered]
In VSCode, press the shortcut key Ctrl+` to open the terminal, which appears on the lower side

Right mouse click moves the panel to the right. I need a setting or configuration to automatically go to the right every time I open it

**Comment by alefragnani** [comment, marked as answer]
Hi @dong-lufei ,

There is already a setting to define _where the entire panel_ will be located (`"workbench.panel.defaultLocation"` - _Controls the default location of the panel (Terminal, Debug Console, Output, Problems) in a new workspace. It can either show at the bottom, right, or left of the editor area._). 

If this setting doesn't fit your needs, I suggest you to open a new feature request, directly in the VS Code repo (https://github.com/microsoft/vscode/issues/new/choose), which is the appropriate location for such request.

This repository, on the other hand, is a place to discuss VS Code's extension platform

Hope this helps

**Reply by dong-lufei** [reply]
After asking the question, go back and find this setting on the interface. Yes, sure
